src/components/PieChartComponent.jsx

```diff
@@ -1,7 +1,4 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import FusionCharts from 'fusioncharts';
-import Charts from 'fusioncharts/fusioncharts.charts';
```

Comment:
Are all the deletions in this file unused imports?

Reply:
They were not being used when the dashboard loads up. The terminal would flag them as `no-unused-vars`